CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+* Update PowerSTIG to Parse/Apply Red Hat Enterprise Linux 7 STIG V3R8: [#1179](https://github.com/microsoft/PowerStig/issues/1179)
```

Comment:
I think this is a typo should be V3R9

Reply:
Update PowerSTIG to Parse/Apply Red Hat Enterprise Linux 7 STIG V3R8: this still exists
should be v2R9